app/uk/gov/hmrc/helptosavefrontend/controllers/AccountHolderController.scala

```diff
@@ -133,7 +133,13 @@ class AccountHolderUpdateEmailAddressController @Inject() (val helpToSaveService
       },
       email ⇒ Ok(views.html.email.we_updated_your_email(email))
     )
-  }(redirectOnLoginURL = routes.AccountHolderUpdateEmailAddressController.getEmailVerified.url)
+  }(redirectOnLoginURL = routes.AccountHolderController.getEmailVerified.url)
+
+  def getCloseAccountPage: Action[AnyContent] = authorisedForHtsWithNINO { implicit request ⇒ implicit htsContext ⇒
```

Comment:
do we need `authorisedForHtsWithNINO ` can we use `authorisedForHts` instead here ?

Reply:
I am using authorisedForHtsWithNINO so I can checkIfAlreadyEnrolled, is this check necessary, will a user who is not yet enrolled be able to access this page?